### PR TITLE
fix: FGA enablement

### LINKED ISSUE #21515: FGA regression backports
Backports https://github.com/influxdata/influxdb/issues/21514

- [x] 1.8 branch
- [x] 1.9 branch

### PR DESCRIPTION
**Pull request by lesam**
Closes #21515

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: services/httpd/handler.go [hunk 1]
```diff
@@ -583,6 +585,8 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user meta.U
 			h.httpError(rw, "error authorizing query: "+err.Error(), http.StatusForbidden)
```
Comment (davidby-influx) [inline services/httpd/handler.go:585]: Do we want a log event in the omitted else branch to log the fact we can't get a fineAuthorizer?  Seems like something we would want to report, independently of returning it as a StatusForbidden error.

File: services/meta/query_authorizer.go [hunk 2]
```diff
@@ -132,7 +133,10 @@ func (a *QueryAuthorizer) AuthorizeDatabase(u User, priv influxql.Privilege, dat
 		User:     u.ID(),
 		Message:  fmt.Sprintf("Internal error - incorrect oss user type %T", u),
 	}
+}
 
+func (a *QueryAuthorizer) FineAuthorizer(u User) (query.FineAuthorizer, error) {
```
Comment (davidby-influx) [inline services/meta/query_authorizer.go:138]: Is this correct?  Where is it used?

**Review by davidby-influx — CHANGES_REQUESTED** [review]
A few questions.

File: services/httpd/handler.go (hunk 1 above)
Comment (lesam) [reply services/httpd/handler.go:587]: Done

File: services/meta/query_authorizer.go (hunk 2 above)
Comment (lesam) [reply services/meta/query_authorizer.go]: Unused cruft - thanks.

**Review by davidby-influx — APPROVED** [review]
Looks good, previous issues addressed.

**Pull request merged**